src/var/getShadowRoot.js

```diff
@@ -0,0 +1,15 @@
+define( function() {
+	"use strict";
+
+	return function getShadowRoot( obj ) {
```

Comment:
For clarity, please update both this function and `isAttached` to use `elem` instead of `obj`.